test/unit/lib/getPharmacies.js

```diff
@@ -47,14 +48,48 @@ describe('Nearby', () => {
       });
     });
 
-    it('should get the number of results requested, ordered by distance', () => {
-      const requestedNumberOfResults = 5;
-      const results = pharmacies.nearby(searchPoint, geo, requestedNumberOfResults).nearbyServices;
+    it('should get the number of unique nearby open services requested, ordered by distance',
+      () => {
+        const requestedNumberOfOpenResults = 4;
+        const results =
+          pharmacies
+          .nearby(sillySearchPoint, geo, { open: requestedNumberOfOpenResults })
+          .openServices;
+
+        expect(results.length).to.be.equal(requestedNumberOfOpenResults);
+
+        const identifiers = [];
+        let previousDistance = 0;
+
+        results.forEach((result) => {
+          const identifier = result.identifier;
+          expect(identifiers.includes(identifier))
+            .to.be.equal(false, `${identifier} is contained in the list of results already`);
+          identifiers.push(identifier);
+
+          expect(result.distanceInMiles).to.be.at.least(previousDistance);
+          previousDistance = result.distanceInMiles;
+        });
+      });
+
+    it('should get the number of unique nearby services requested, ordered by distance', () => {
+      const requestedNumberOfResults = 52;
+      const results =
+        pharmacies
+        .nearby(sillySearchPoint, geo, { nearby: requestedNumberOfResults })
```

Comment:
Ah, now I know why. It's a 'Scilly' point. Still not sure why it is being used over search point. Suspect it is to do with remoteness. Maybe rename?